src/Message/AbstractRequest.php

```diff
@@ -1139,17 +1139,14 @@ protected function buildPaymentMethod($paymentMethodType, $addAttributes = false
             if (!$this->isUpdate()) {
                 $paymentMethod->type = self::PAYMENT_METHOD_PAYPAL;
             }
-        } elseif ($paymentMethodType === self::PAYMENT_METHOD_APPLE_PAY && $card !== null) {
+        } elseif ($paymentMethodType === self::PAYMENT_METHOD_APPLE_PAY) {
```

Comment:
Still testing to be 100% sure, don't believe the card param is necessary.

Reply:
What's the update on this, was it actually no necessary?

Reply:
I was just testing this. When we don't include the card param with a billing address. Vindicia returns an okay (200 'OK'), however, sales tax isn't calculated. We get this error in name values for taxable items:

`"Can't compute sales tax due to lack of shipping and billing addresses"`

So I'll add `&& $card` to this line and update necessary documentation.

Reply:
cool 👍